packages/design-tokens/design-token-previews/utils/previews.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+
+import includes from "lodash/includes";
+
+import { TokenEntry } from "../types/TokenEntry";
+
+import { FontSizePreview } from "../FontSizePreview";
+import { FontWeightPreview } from "../FontWeightPreview";
+import { IconSizePreview } from "../IconSizePreview";
+
+export const createFontWeightPreview = ([
+  tokenSuffix,
```

Comment:
Does it make sense to create components for things you are not reusing?
If you want to have review components I would create one per type. e.g., reusing font size and weight as they render the same content (we change just the prop).
Same for borders. We are just changing the prop.


Rethinking it. I think we need only;y for icons. The rest could be all the same.

Reply:
https://github.com/Localitos/pluto/pull/988/files#r1321781393